mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -163,50 +183,59 @@ def ocn_modelvsobs(config, field):
         monthsvalue = constants.monthdictionary[timestring]
 
         if isinstance(monthsvalue, (int, long)):
-            modeldata = monthly_clim.sel(month=monthsvalue).mpasData.values
-            obsdata = dsData.sel(month = monthsvalue).observationData.values
+            modeldata = monthly_clim.sel(month=monthsvalue)[field].values
+            obsdata = dsData.sel(month=monthsvalue)[obsFieldName].values
         else:
-            modeldata = np.sum(constants.dinmonth[monthsvalue-1]*monthly_clim.sel(month=monthsvalue).
-        		               mpasData.values.T, axis=1) / np.sum(constants.dinmonth[monthsvalue-1])
-            obsdata = np.nansum(daysarray[monthsvalue-1, :, :]*dsData.sel(month = monthsvalue).
-                                observationData.values, axis=0) / np.nansum(daysarray[monthsvalue-1, :, :],
-                                axis=0)
+
+            modeldata = np.sum(
+                constants.dinmonth[monthsvalue-1] *
+                monthly_clim.sel(month=monthsvalue)[field].values.T,
+                axis=1) / np.sum(constants.dinmonth[monthsvalue-1])
```

Comment:
Okay, I figured out a way.  Technically, though I think the visual problem is not that the axis=1 needs to be on the previous line but that the division should *not* be on the same line.  But whatever, I fixed it...